astropy/coordinates/transformations/composite.py

```diff
@@ -0,0 +1,259 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""Module defining the class for a composited sequence of coordinate transformations.
+
+This module contains the class
+:class:`~astropy.coordinates.transformations.CompositeTransform` that combines a
```

Comment:
I'd remove `.transformations` in the link.

Reply:
✅